2022-10-29 10:50:21,846 INFO    MainThread:17340 [wandb_setup.py:_flush():68] Configure stats pid to 17340
2022-10-29 10:50:21,846 INFO    MainThread:17340 [wandb_setup.py:_flush():68] Loading settings from C:\Users\marka\.config\wandb\settings
2022-10-29 10:50:21,846 INFO    MainThread:17340 [wandb_setup.py:_flush():68] Loading settings from C:\Users\marka\Documents\cardiovascular-disease-detection\wandb\settings
2022-10-29 10:50:21,846 INFO    MainThread:17340 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-10-29 10:50:21,846 INFO    MainThread:17340 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-10-29 10:50:21,847 INFO    MainThread:17340 [wandb_init.py:_log_setup():476] Logging user logs to C:\Users\marka\Documents\cardiovascular-disease-detection\wandb\run-20221029_105021-dj7a8ubb\logs\debug.log
2022-10-29 10:50:21,847 INFO    MainThread:17340 [wandb_init.py:_log_setup():477] Logging internal logs to C:\Users\marka\Documents\cardiovascular-disease-detection\wandb\run-20221029_105021-dj7a8ubb\logs\debug-internal.log
2022-10-29 10:50:21,847 INFO    MainThread:17340 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001EB4CED78B0>
2022-10-29 10:50:21,847 INFO    MainThread:17340 [wandb_init.py:init():516] calling init triggers
2022-10-29 10:50:21,847 INFO    MainThread:17340 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {'epochs': 100, 'batch_size': 128}
2022-10-29 10:50:21,847 INFO    MainThread:17340 [wandb_init.py:init():569] starting backend
2022-10-29 10:50:21,847 INFO    MainThread:17340 [wandb_init.py:init():573] setting up manager
2022-10-29 10:50:21,851 INFO    MainThread:17340 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn, using: spawn
2022-10-29 10:50:21,857 INFO    MainThread:17340 [wandb_init.py:init():580] backend started and connected
2022-10-29 10:50:21,868 INFO    MainThread:17340 [wandb_run.py:_label_probe_notebook():1113] probe notebook
2022-10-29 10:50:21,870 INFO    MainThread:17340 [wandb_run.py:_label_probe_notebook():1123] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-10-29 10:50:21,870 INFO    MainThread:17340 [wandb_init.py:init():658] updated telemetry
2022-10-29 10:50:21,932 INFO    MainThread:17340 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2022-10-29 10:50:22,464 INFO    MainThread:17340 [wandb_run.py:_on_init():2000] communicating current version
2022-10-29 10:50:22,631 INFO    MainThread:17340 [wandb_run.py:_on_init():2004] got version response 
2022-10-29 10:50:22,631 INFO    MainThread:17340 [wandb_init.py:init():728] starting run threads in backend
2022-10-29 10:50:27,653 INFO    MainThread:17340 [wandb_run.py:_console_start():1980] atexit reg
2022-10-29 10:50:27,653 INFO    MainThread:17340 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.WRAP_RAW
2022-10-29 10:50:27,654 INFO    MainThread:17340 [wandb_run.py:_redirect():1903] Wrapping output streams.
2022-10-29 10:50:27,654 INFO    MainThread:17340 [wandb_run.py:_redirect():1925] Redirects installed.
2022-10-29 10:50:27,654 INFO    MainThread:17340 [wandb_init.py:init():765] run started, returning control to user process
2022-10-29 10:58:36,764 INFO    MainThread:17340 [wandb_init.py:_pause_backend():389] pausing backend
2022-10-29 10:58:36,764 INFO    MainThread:17340 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-10-29 11:06:41,180 INFO    MainThread:17340 [wandb_init.py:_resume_backend():398] resuming backend
2022-10-29 11:06:41,277 INFO    MainThread:17340 [wandb_init.py:_pause_backend():389] pausing backend
2022-10-29 11:06:41,277 INFO    MainThread:17340 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-10-29 11:06:54,292 INFO    MainThread:17340 [wandb_init.py:_resume_backend():398] resuming backend
2022-10-29 11:06:54,294 INFO    MainThread:17340 [wandb_init.py:_pause_backend():389] pausing backend
2022-10-29 11:06:54,294 INFO    MainThread:17340 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-10-29 11:07:17,995 INFO    MainThread:17340 [wandb_init.py:_resume_backend():398] resuming backend
2022-10-29 11:07:18,306 INFO    MainThread:17340 [wandb_init.py:_pause_backend():389] pausing backend
2022-10-29 11:07:18,306 INFO    MainThread:17340 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-10-29 11:07:33,571 INFO    MainThread:17340 [wandb_init.py:_resume_backend():398] resuming backend
2022-10-29 11:07:33,592 INFO    MainThread:17340 [wandb_setup.py:_flush():68] Configure stats pid to 17340
2022-10-29 11:07:33,592 INFO    MainThread:17340 [wandb_setup.py:_flush():68] Loading settings from C:\Users\marka\.config\wandb\settings
2022-10-29 11:07:33,592 INFO    MainThread:17340 [wandb_setup.py:_flush():68] Loading settings from C:\Users\marka\Documents\cardiovascular-disease-detection\wandb\settings
2022-10-29 11:07:33,592 INFO    MainThread:17340 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-10-29 11:07:33,592 INFO    MainThread:17340 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-10-29 11:07:33,593 INFO    MainThread:17340 [wandb_init.py:_log_setup():476] Logging user logs to C:\Users\marka\Documents\cardiovascular-disease-detection\wandb\run-20221029_110733-s2q5g15t\logs\debug.log
2022-10-29 11:07:33,593 INFO    MainThread:17340 [wandb_init.py:_log_setup():477] Logging internal logs to C:\Users\marka\Documents\cardiovascular-disease-detection\wandb\run-20221029_110733-s2q5g15t\logs\debug-internal.log
2022-10-29 11:07:33,593 INFO    MainThread:17340 [wandb_init.py:init():516] calling init triggers
2022-10-29 11:07:33,593 INFO    MainThread:17340 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {'epochs': 100, 'batch_size': 64}
2022-10-29 11:07:33,593 INFO    MainThread:17340 [wandb_init.py:init():535] re-initializing run, found existing run on stack: dj7a8ubb
2022-10-29 11:07:33,595 INFO    MainThread:17340 [wandb_run.py:_finish():1746] finishing run markalsa/cardiovascular-disease-detection/dj7a8ubb
2022-10-29 11:07:33,595 INFO    MainThread:17340 [jupyter.py:save_history():448] not saving jupyter history
2022-10-29 11:07:33,595 INFO    MainThread:17340 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-10-29 11:07:33,595 INFO    MainThread:17340 [wandb_init.py:_jupyter_teardown():408] cleaning up jupyter logic
2022-10-29 11:07:33,596 INFO    MainThread:17340 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-10-29 11:07:33,596 INFO    MainThread:17340 [wandb_run.py:_restore():1932] restore
2022-10-29 11:07:33,596 INFO    MainThread:17340 [wandb_run.py:_restore():1938] restore done
2022-10-29 11:07:39,722 INFO    MainThread:17340 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-10-29 11:07:39,724 INFO    MainThread:17340 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2022-10-29 11:07:39,731 INFO    MainThread:17340 [wandb_run.py:_footer_sync_info():3333] logging synced files
